docs/Makefile

```diff
@@ -1,3 +1,10 @@
 clean:
 	rm -f src/sqlitegen/all.html src/sqlitegen/syntax_linkage.tcl
 
+
+spelings:
```

Comment:
This is one of those "immediately obvious in hindsight" things that I should have done.  Thanks Patrick.